htdocs/langs/fr_FR/bills.lang

```diff
@@ -1,4 +1,4 @@
-# Dolibarr language file - Source file is en_US - bills
+<# Dolibarr language file - Source file is en_US - bills
```

Comment:
Change on non en_us fikes must not be included into pr

Reply:
It's done

Reply:
It's done